about/system_requirements.rst

```diff
@@ -265,15 +270,19 @@ Mobile device (smartphone/tablet) - Minimum
 |                      |                                                                                         |
 |                      |   - *Example: Apple A7 (iPhone 5S)*                                                     |
 +----------------------+-----------------------------------------------------------------------------------------+
-| **GPU**              | - **Forward+ rendering method:** SoC featuring GPU with full Vulkan 1.0 support         |
+| **GPU**              | - **Forward+ renderer:** SoC featuring GPU with full Vulkan 1.0 support, or             |
+|                      |   Metal 3 support (iOS/iPadOS)                                                          |
 |                      |                                                                                         |
-|                      |   - *Example: Qualcomm Adreno 505, Mali-G71 MP2, Apple A12 (iPhone XR/XS)*              |
+|                      |   - *Example (Vulkan): Qualcomm Adreno 505, Mali-G71 MP2, Apple A12 (iPhone XR/XS)*     |
+|                      |   - *Example (Metal): Apple A13 (iPhone 11/SE 2nd generation)*                          |
```

Comment:
We can make this A11 now with https://github.com/godotengine/godot/pull/99820

Reply:
Updated to A11.